core-api/src/main/java/com/optimizely/ab/event/internal/EventFactory.java

```diff
@@ -168,6 +168,16 @@ public LogEvent createConversionEvent(@Nonnull ProjectConfig projectConfig,
         List<Attribute> attributesList = new ArrayList<Attribute>();
 
         for (Map.Entry<String, ?> entry : attributes.entrySet()) {
+            // Filters down to the types of values that are allowed.
```

Comment:
Can we link to https://github.com/optimizely/avro-schemas/blob/baeba19a7bd227aae67026474aec239110c5801d/event_batch/src/main/avro/EventBatch.avsc#L22 since that's the source of truth? Or actually https://developers.optimizely.com/x/events/api/#Attribute_ might be preferable since not everyone can view the avro-schemas repo